packages/protocol/test/governance/release_gold.ts

```diff
@@ -1062,86 +1062,88 @@ contract('ReleaseGold', (accounts: string[]) => {
-            assertEqualBN(releasedBalanceAtRevoke, 0)
+          describe('when called by releaseOwner', () => {
+            it('should succeed', async () => {
+              await releaseGoldInstance.expire({ from: releaseOwner })
+            })
           })
 
-          it('should allow refund of all remaining gold', async () => {
-            const refundAddressBalanceBefore = await goldTokenInstance.balanceOf(refundAddress)
-            await releaseGoldInstance.refundAndFinalize({ from: releaseOwner })
-            const refundAddressBalanceAfter = await goldTokenInstance.balanceOf(refundAddress)
-            assertEqualBN(refundAddressBalanceAfter.minus(refundAddressBalanceBefore), TOTAL_AMOUNT)
-          })
-        })
+          describe('when an instance is expired', () => {
+            beforeEach(async () => {
+              await releaseGoldInstance.expire({ from: releaseOwner })
+            })
 
-        describe('when an instance is a registered validator or validator group', () => {
-          describe('registered validator', () => {
-            it('should revert', async () => {
-              await mockValidators.setValidator(releaseGoldInstance.address)
-              await assertRevert(releaseGoldInstance.expire({ from: releaseOwner }))
+            it('should revoke the contract', async () => {
+              const isRevoked = await releaseGoldInstance.isRevoked()
+              assert.equal(isRevoked, true)
             })
-          })
-          describe('registered validator', () => {
-            it('should revert', async () => {
-              await mockValidators.setValidatorGroup(releaseGoldInstance.address)
-              await assertRevert(releaseGoldInstance.expire({ from: releaseOwner }))
+
+            it('should set the released balance at revocation to total withdrawn', async () => {
+              const [, , releasedBalanceAtRevoke] = await releaseGoldInstance.revocationInfo()
+              // 0 gold withdrawn at this point
+              assertEqualBN(releasedBalanceAtRevoke, 0)
+            })
+
+            it('should allow refund of all remaining gold', async () => {
+              const refundAddressBalanceBefore = await goldTokenInstance.balanceOf(refundAddress)
+              await releaseGoldInstance.refundAndFinalize({ from: releaseOwner })
+              const refundAddressBalanceAfter = await goldTokenInstance.balanceOf(refundAddress)
+              assertEqualBN(
+                refundAddressBalanceAfter.minus(refundAddressBalanceBefore),
+                TOTAL_AMOUNT
+              )
             })
           })
         })
       })
     })
+
+    describe('when the contract is not expirable', () => {
+      beforeEach(async () => {
+        await createNewReleaseGoldInstance(releaseGoldDefaultSchedule, web3)
+        await releaseGoldInstance.setCanExpire(false, { from: beneficiary })
+      })
+
+      describe('when `expire` is called', () => {
```

Comment:
Can we time travel to the point where it *would* be expirable if the boolean was set? Otherwise we're testing a different condition here